File: python/mirage/accessor/wandb.py
from mirage.accessor.base import SessionAccessor
from mirage.core.wandb.client import WandbClient
from mirage.core.wandb.config import WandbConfig


class WandbAccessor(SessionAccessor):

    def __init__(self, config: WandbConfig) -> None:
        super().__init__()
        self.config = config
        self.client = WandbClient(config, self.pool)

File: typescript/packages/core/src/accessor/wandb.ts
import { Accessor } from './base.ts'
import { WandbClient } from '../core/wandb/client.ts'
import type { WandbConfig } from '../core/wandb/config.ts'
export class WandbAccessor extends Accessor {
  readonly client: WandbClient
  constructor(readonly config: WandbConfig) {
    super()
    this.client = new WandbClient(config)
  }
}
